From dcf63cade48f7f2fbdcd8392f4950f565ef95c20 Mon Sep 17 00:00:00 2001
From: Chris Morgan <macromorgan@hotmail.com>
Date: Wed, 30 Oct 2024 20:40:53 -0500
Subject: [PATCH] arm64: dts: rockchip: Add avdd_0v9 and avdd_1v8 for RGxx3
 HDMI

Define the correct 0.9v and 1.8v regulators for the RGxx3 HDMI port.
The regulators used here appear to power most of the 0.9v and 1.8v
power groups listed on table 3-1 of the rk3566 datasheet.

Signed-off-by: Chris Morgan <macromorgan@hotmail.com>
---
 arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
index b9018bed29de..3d30adbec8c1 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
@@ -177,6 +177,8 @@ &hdmi {
 	ddc-i2c-bus = <&i2c5>;
 	pinctrl-0 = <&hdmitxm0_cec>;
 	pinctrl-names = "default";
+	avdd-0v9-supply = <&vdda0v9_pmu>;
+	avdd-1v8-supply = <&vcca1v8_pmu>;
 	status = "okay";
 };
 
-- 
2.34.1

